tools/sidekick/app.js

```diff
@@ -331,23 +348,21 @@
       id: 'preview',
       condition: (sidekick) => sidekick.isEditor() || sidekick.isOuter(),
       button: {
-        action: () => {
+        action: (evt) => {
           const { config, location } = sk;
           let url;
           if (sk.isEditor()) {
-            url = new URL('https://adobeioruntime.net/api/v1/web/helix/helix-services/content-proxy@v2');
-            url.search = new URLSearchParams([
-              ['owner', config.owner],
-              ['repo', config.repo],
-              ['ref', config.ref || 'main'],
-              ['path', '/'],
-              ['lookup', location.href],
-            ]).toString();
+            const previewPath = `/hlx_${btoa(location.href).replace(/\+/, '-').replace(/\//, '_')}.lnk`;
```

Comment:
What are the 2 `replace(...)` doing on the b64 encoded href ?

Reply:
They're to keep the b64 safe for use in a URL.

Reply:
ah! I did not know you can get a `+` or a `/` in b64 :)

Reply:
Maybe this is obsolete... tbh I took it over from a NodeJS implementation by @tripodsan 😄

Reply:
`btoa` still uses the original set:

> The algorithm used by atob() and btoa() is specified in RFC 4648, section 4.

```
     Value Encoding  Value Encoding  Value Encoding  Value Encoding
         0 A            17 R            34 i            51 z
         1 B            18 S            35 j            52 0
         2 C            19 T            36 k            53 1
         3 D            20 U            37 l            54 2
         4 E            21 V            38 m            55 3
         5 F            22 W            39 n            56 4
         6 G            23 X            40 o            57 5
         7 H            24 Y            41 p            58 6
         8 I            25 Z            42 q            59 7
         9 J            26 a            43 r            60 8
        10 K            27 b            44 s            61 9
        11 L            28 c            45 t            62 +
        12 M            29 d            46 u            63 /
        13 N            30 e            47 v
        14 O            31 f            48 w         (pad) =
        15 P            32 g            49 x
        16 Q            33 h            50 y
```